runtime/vara/src/lib.rs

```diff
@@ -1228,7 +1228,10 @@ pub type BuiltinActors = (
 pub type BuiltinActors = (
     ActorWithId<1, pallet_gear_builtin::bls12_381::Actor<Runtime>>,
     ActorWithId<2, pallet_gear_builtin::staking::Actor<Runtime>>,
-    ActorWithId<3, pallet_gear_eth_bridge::Actor<Runtime>>,
+    ActorWithId<
```

Comment:
this might be reverted as well, but off top: this import looks good so we may consider importing constant from Actors modules itself

Reply:
Fixed